README.md

```diff
@@ -30,14 +32,16 @@ A little table to coordinate the midstream release/ci automation for our fork re
 | 2021-10-04T01:00:00Z | devguyio |
 | 2021-10-11T01:00:00Z | aliok    |
 | 2021-10-18T01:00:00Z | lberk    |
-| 2021-10-25T01:00:00Z | aliok    |
+| 2021-10-25T01:00:00Z | pierdipi |
```

Comment:
Replaced Ali since he will do it just 2 weeks before this.

Reply:
I suggest this should be like this because @matzew is replacing @aliok for the 11.10 week

|2021-10-11T01:00:00Z | matzew |
| 2021-10-18T01:00:00Z | aliok |
| 2021-10-25T01:00:00Z | lberk |
| 2021-11-01T01:00:00Z | pierdipi |
| 2021-11-08T01:00:00Z | devguyio |

Reply:
Ok, then I can replace Ali next week (11.10) and leave the other one as they are, wdyt?